addons/utm/models/utm_campaign.py

```diff
@@ -20,7 +20,7 @@ class UtmCampaign(models.Model):
     stage_id = fields.Many2one(
```

Comment:
Commit message:

- You say twice "add an avatar widget for the Responsible field".
- Github shows a lot of spacing for the "embed bullet points"

Reply:
![image](https://user-images.githubusercontent.com/41282536/232014276-998402bf-a368-4ed7-99e7-a7e9a2739fe0.png)

Was talking about the commit message, sorry if it was unclear :slightly_smiling_face: